Jenkinsfile

```diff
@@ -33,13 +33,6 @@ pipeline {
         disableConcurrentBuilds()
         timestamps()
     }
-    triggers {
-        /*
-          Restrict nightly builds to master branch, all others will be built on change only.
-          Note: The BRANCH_NAME will only work with a multi-branch job using the github-branch-source
-        */
-        cron(env.BRANCH_NAME == "master" ? "H H(21-23) * * *" : "")
-    }
```

Comment:
Reminder to remove the PR configuration on Jenkins:
![image](https://user-images.githubusercontent.com/8041246/63297304-f2b68e80-c285-11e9-896f-c0277592a9b7.png)

Reply:
Done